plugins/serializers/nowmetric/nowmetric.go

```diff
@@ -0,0 +1,122 @@
+package nowmetric
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/influxdata/telegraf"
+)
+
+type serializer struct {
+	TimestampUnits time.Duration
+}
+
+const METRICFMT string = "{ \"metric_type\": \"%s\", \"resource\": \"%s\", \"node\": \"%s\", \"value\": %v, \"timestamp\": %d, \"ci2metric_id\": { \"node\": \"%s\" }, \"source\": \"Telegraf\" }"
```

Comment:
This is just a quick lookthrough, not a full review, but this needs to be changed to use the json.Marshal function on a struct or array of structs, as this won't do suitable escaping.